src/graphics/Explain.ts

```diff
@@ -69,320 +69,319 @@ export const EXPLAIN_DEATH = 'Surviving Death';
-        visible: false,
-        complete: false,
-        text: "Cast more than once in a single turn",
-        nextVisibleTasks: [],
-        showExplainPopup: [],
-    },
-    camera: {
-        visible: false,
-        complete: false,
-        text: "Move the camera with W,A,S, and D keys or by holding and dragging Middle Mouse Button",
-        nextVisibleTasks: ['recenterCamera'],
-        showExplainPopup: [],
-    },
-    recenterCamera: {
-        visible: false,
-        complete: false,
-        text: "Recenter the camera with the Z key",
-        nextVisibleTasks: [],
-        showExplainPopup: [],
-    },
+  spawn: {
+    visible: true,
+    complete: false,
+    text: "Click somewhere on the grass to choose a spawn point",
+    nextVisibleTasks: ['moved', 'portal'],
+    showExplainPopup: [],
+  },
+  moved: {
+    visible: false,
+    complete: false,
+    text: "Hold right mouse button to walk towards your cursor",
+    nextVisibleTasks: [],
+    showExplainPopup: [EXPLAIN_WALK],
+  },
+  portal: {
+    visible: false,
+    complete: false,
+    text: "Move into the portal to go to the next level",
+    nextVisibleTasks: ['camera', 'cast'],
+    showExplainPopup: [],
+  },
+  cast: {
+    visible: false,
+    complete: false,
+    text: "Click on a spell from your toolbar to queue it up and then click on a target to cast it",
+    nextVisibleTasks: ['castMultipleInOneTurn'],
+    showExplainPopup: [EXPLAIN_CAST],
+  },
+  castMultipleInOneTurn: {
+    visible: false,
+    complete: false,
+    text: "Cast more than once in a single turn",
+    nextVisibleTasks: [],
+    showExplainPopup: [],
+  },
+  camera: {
+    visible: false,
+    complete: false,
+    text: "Move the camera with W,A,S, and D keys or by holding and dragging Middle Mouse Button",
+    nextVisibleTasks: ['recenterCamera'],
+    showExplainPopup: [],
+  },
+  recenterCamera: {
+    visible: false,
+    complete: false,
+    text: "Recenter the camera with the Z key",
+    nextVisibleTasks: [],
+    showExplainPopup: [],
+  },
 }
 
 export function updateTutorialChecklist() {
-    if (globalThis.usingTestRunner) {
-        return;
+  if (globalThis.usingTestRunner) {
+    return;
+  }
+  let html = `<h3>${i18n('Tutorial')}</h3>`;
+  const tutorialItems = Object.values(tutorialChecklist);
+  const completeTutorialItems = tutorialItems.filter(x => x.complete);
+  if (completeTutorialItems.length) {
+    let completedItemsHtml = '';
+    for (let item of completeTutorialItems) {
+      completedItemsHtml += `<div class="complete">&#x2611; <span class="text complete">${i18n(item.text)}</span></div>`
     }
-    let html = `<h3>${i18n('Tutorial')}</h3>`;
-    const tutorialItems = Object.values(tutorialChecklist);
-    const completeTutorialItems = tutorialItems.filter(x => x.complete);
-    if (completeTutorialItems.length) {
-        let completedItemsHtml = '';
-        for (let item of completeTutorialItems) {
-            if (item.visible) {
```

Comment:
Why was this removed?

Reply:
> Fixed a bug that prevented completed tutorial tasks from appearing in the completed task dropdown

This just makes it so that all completed tutorial tasks are shown in the dropdown when you click on it. Previously it would have said 4/7 tasks complete, but the list would only have 1 or 2 completed tasks, which felt buggy.